src/main/java/uk/gov/justice/digital/delius/data/api/ContactableHuman.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static org.apache.commons.text.WordUtils.capitalizeFully;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class ContactableHuman {
+    @ApiModelProperty(value = "Given names", example = "Sheila Linda")    
+    private String forenames;
+    @ApiModelProperty(value = "Family name", example = "Hancock")    
+    private String surname;
+    @ApiModelProperty(value = "Email id", example = "officer@gov.uk")
```

Comment:
would be better as `"Email address"` instead of `"Email id"`

Reply:
fixed